packages/discovery-provider/src/queries/comments/replies.py

```diff
@@ -0,0 +1,114 @@
+import logging
+
+from src.api.v1.helpers import format_limit, format_offset
+from src.models.comments.comment import Comment
+from src.models.tracks.track import Track
+from src.queries.comments.utils import (
+    COMMENT_REPLIES_DEFAULT_LIMIT,
+    _format_comment_response,
+    build_comments_query,
+    get_base_comments_query,
+)
+from src.utils.db_session import get_db_read_replica
+
+logger = logging.getLogger(__name__)
+
+
+def get_replies(
+    session,
+    parent_comment_id,
+    current_user_id,
+    # note: artist id already exists when used via get_track_comments - no need to requery for it
+    artist_id=None,
+    offset=0,
+    limit=COMMENT_REPLIES_DEFAULT_LIMIT,
+):
```

Comment:
Similarly, wondering how hard it would be to type the output of this query. I've been trying to add types to new code in discovery when possible.